packages/tpetra/core/test/LinearProblem/LinearProblem_UnitTests.cpp

```diff
@@ -0,0 +1,297 @@
+// @HEADER
+// *****************************************************************************
+//          Tpetra: Templated Linear Algebra Services Package
+//
+// Copyright 2008 NTESS and the Tpetra contributors.
+// SPDX-License-Identifier: BSD-3-Clause
+// *****************************************************************************
+// @HEADER
+
+#include "Tpetra_TestingUtilities.hpp"
+#include "Tpetra_LinearProblem.hpp"
+
+#include "Tpetra_CrsMatrix.hpp"
+
+
+namespace { // (anonymous)
+
+  using Tpetra::TestingUtilities::getDefaultComm;
+  using Tpetra::createContigMapWithNode;
+  using Teuchos::RCP;
+  using Teuchos::rcp;
+  using Teuchos::Comm;
+  using Teuchos::Array;
+  using Teuchos::tuple;
+  //using Teuchos::NO_TRANS;
+  //using Teuchos::TRANS;
+  //using Teuchos::CONJ_TRANS;
+  using std::endl;
+  using GST = Tpetra::global_size_t;
+  
+
+  /// \brief Print out pretty version of RowMatrix.
```

Comment:
Can all the display utilities be commented or macro'd out, since their use is also commented out below?

Reply:
I added a macro.